### PR TITLE
Fix Sphinx docstring formatting error. (#13004, #13005, #13006)

### LINKED ISSUE #13004: mxnet.gluon.rnn.GRUCell docs error
## Description
Sphinx throws an error generating docs for `mxnet.gluon.rnn.GRUCell`.

## Error
```
/home/ubuntu/incubator-mxnet/python/mxnet/gluon/rnn/__init__.py:docstring of mxnet.gluon.rnn.GRUCell:: ERROR: Unknown target name: "gru".
```

### LINKED ISSUE #13005: mxnet.gluon.rnn.LSTMCell docs error
## Description
Sphinx throws an error when generating docs for `mxnet.gluon.rnn.LSTMCell`.

## Error
```
/home/ubuntu/incubator-mxnet/python/mxnet/gluon/rnn/__init__.py:docstring of mxnet.gluon.rnn.LSTMCell:: ERROR: Unknown target name: "lstm".
```

### LINKED ISSUE #13006: mxnet.gluon.rnn.RNNCell docs error
## Description
Sphinx throws an error when generating docs for `mxnet.gluon.rnn.RNNCell`.

## Error
```
/home/ubuntu/incubator-mxnet/python/mxnet/gluon/rnn/__init__.py:docstring of mxnet.gluon.rnn.RNNCell:: ERROR: Unknown target name: "rnn".
```

### PR DESCRIPTION
**Pull request by frankfliu**
## Description ##
Use inline literal to quote _, avoid Sphinx considering as link
Fixes #13004, Fixes #13005, Fixes #13006

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [X] Changes are complete (i.e. I finished coding on this PR)
- [X] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [X] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [X] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### CODE REVIEW

**Comment by anirudhacharya** [comment]
@mxnet-label-bot [pr-awaiting-review, Doc]

File: python/mxnet/gluon/rnn/rnn_cell.py [hunk 1]
```diff
@@ -333,7 +333,7 @@ class RNNCell(HybridRecurrentCell):
         Initializer for the bias vector.
     h2h_bias_initializer : str or Initializer, default 'zeros'
         Initializer for the bias vector.
-    prefix : str, default 'rnn_'
+    prefix : str, default ```'rnn_'```
```
Comment (nswamy) [inline python/mxnet/gluon/rnn/rnn_cell.py]: Usually we ``` for a block and for single variable we use `. Did that fail?

Comment (nswamy) [inline python/mxnet/gluon/rnn/rnn_cell.py]: Usually we ``` for a block and for single variable we use `. Did that fail?

Comment (nswamy) [reply python/mxnet/gluon/rnn/rnn_cell.py]: discussed offline, @frankfliu showed that a single ` turns into a link.
http://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-the-linkcheck-builder

@frankfliu will change this to ``

**Pull request merged**